src/components/SharedComponents/BottomSheet.js

```diff
@@ -1,53 +1,53 @@
 // @flow
 
-import {
-  BottomSheetBackdrop,
-  BottomSheetModal,
-  BottomSheetModalProvider
-} from "@gorhom/bottom-sheet";
+import BottomSheet, { BottomSheetView } from "@gorhom/bottom-sheet";
 import type { Node } from "react";
-import React, { useEffect, useRef } from "react";
+import React, {
+  useCallback, useEffect, useMemo, useRef
+} from "react";
 
 import { viewStyles } from "../../styles/sharedComponents/bottomSheet";
 
 type Props = {
-  children: any
+  children: any,
+  hasScrolled: boolean
 }
 
-const BottomSheet = ( { children }: Props ): Node => {
-  // ref
-  const bottomSheetModalRef = useRef( null );
-
-  const renderBackdrop = props => (
-    <BottomSheetBackdrop
-      // eslint-disable-next-line react/jsx-props-no-spreading
-      {...props}
-      pressBehavior="close"
-      appearsOnIndex={0}
-      disappearsOnIndex={-1}
-    />
-  );
+const StandardBottomSheet = ( { children, hasScrolled }: Props ): Node => {
+  const sheetRef = useRef( null );
+  const snapPoints = useMemo( () => ["45%"], [] );
```

Comment:
Out of curiosity, I'm assuming your memoizing this so that `BottomSheet` doesn't receive a new array object of snap points every time this gets rendered. Could since the value here doesn't depend on props or context or anything, could you also just make it a constant in this file, e.g. `const SNAP_POINTS = ["45%"];` defined outside of the component? Is there an advantage to `useMemo` here?